mimic/resource.py

```diff
@@ -130,3 +147,70 @@ def get_service_resource(self, request, service_id, region_name):
             # workaround for https://github.com/twisted/klein/issues/56
             return NoResource()
         return service_object
+
+
+class MimicRequest(Request):
```

Comment:
Per @glyph's observation, `MimicRequest` should be inheriting `object` as against `MimicLoggingRequest`